GHzDACs/Cleanup/dac.py

```diff
@@ -936,8 +1024,534 @@ class DacRunner_Build11(DacRunner_Build7):
 class DAC_Build11(DAC_Build7):
     RUNNER_CLASS = DacRunner_Build11
 
+
 fpga.REGISTRY[('DAC', 11)] = DAC_Build11
 
+
+class DacRunner_Build12(DacRunner_Build8):
+    pass
+
+
+class DAC_Build12(DAC_Build8):
+    pass
+
+
+fpga.REGISTRY[('DAC', 12)] = DAC_Build12
+
+
+# Jump table #
+
+class DacRunner_Build15(DacRunner_Build7):
+    # noinspection PyMissingConstructor
```

Comment:
(pycharm complains if you have a derived class `__init__` that doesn't call the base class `__init__`. I think this is a good warning because in many languages (java, c++) this happens automatically. also, I think this is actually a pylint warning (like most pycharm warnings).)